src/main/java/domain/LineGenerator.java

```diff
@@ -0,0 +1,30 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class LineGenerator {
+    private final Random random = new Random();
+    private static final LineGenerator INSTANCE = new LineGenerator();
+
+    public static LineGenerator getInstance() {
+        return INSTANCE;
+    }
+
+    public List<Bridge> generateLadder(Width width) {
```

Comment:
```suggestion
    public List<Bridge> generateLine(Width width) {
```
Ladder > Line 으로 네이밍이 수정되었으니 메서드도 같이 수정해주세요 ~